config/initializers/jbuilder_prettify.rb

```diff
@@ -0,0 +1,26 @@
+require "jbuilder"
+
+class Jbuilder
+  ##
+  # Allows you to set @prettify manually in your .jbuilder files.
+  # Example:
+  #   json.prettify true
+  #   json.prettify false
+  #
+  attr_accessor :prettify
```

Comment:
__[reek]__
* [Attribute] is a writable attribute [:ref](https://github.com/troessner/reek/blob/master/docs/Attribute.md)

<!-- Comment by [SideCI](https://sideci.com) -->